detection-rules/attachment_pdf_invoice.yml

```
@@ -0,0 +1,31 @@
+name: "Attachment: PDF Invoice from first-time sender"
+description: |
+  Detect potential fraudulent invoicing threats in pdf files by analyzing OCR'd text.
+type: "rule"
+severity: "medium"
```

```suggestion
severity: "low"
```

i think we might be surprised by how often orgs do actually get PDF invoices from first-time senders. eg we've seen this for some org aliases like support@, where users reply to Invoice emails with questions